encrypted-media-respec.html

```diff
@@ -2820,79 +2820,96 @@
-                Selecting such an option MAY affect the results of the <a def-id="requestMediaKeySystemAccess"></a> call and/or the license requests that are generated from subsequently generated sessions.
-              </p>
-            </li>
-          </ul>
-        </section>
-
-        <section id="encrypt-identifiers">
-          <h3>Encrypt Identifiers</h3>
-          <p>When exposed to the application - either from a <a def-id="message"></a> event or a message from the server, such as one that is passed to <a def-id="update"></a> - <a def-id="distinctive-identifiers"></a> MUST be encrypted at the message exchange level.
-            The encryption MUST ensure that the ciphertext cannot be used as a proxy for the actual identifier, even given the same plaintext.
-            The CDM MUST verify that the encryption key belongs to a valid license server for its Key System.
+        <section id="allow-persistent-data-cleared">
+          <h3>Allow persistent data to be cleared</h3>
+          <p>
+            Implementations that use persistent data MUST allow the user to clear that data such that it is no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
           </p>
-          <p class="issue">Add more specific text about ensuring the desired privacy properties when encrypting identifiers.</p>
-          <p>This MAY be implemented using a <a href="#server-certificate">server certificate</a>.</p>
-          <p>The license server MUST NOT expose a <a def-id="distinctive-identifier"></a> to any entity other than the CDM that sent it.</p>
-          <p class="note">Specifically, it should not be provided to the application or included unencrypted in messages to the CDM.
-            This can be accomplished by encrypting the identifier or message with the identifier or such that it is only decryptable by that specific CDM.
+          <p>
+            Implementations SHOULD allow users to clear persistent data along with cookies [[!COOKIES]] and other site data.
+            It is RECOMMENDED that users be offered the option to clear persistent data as part of user agent features to clear browsing history.
           </p>
-          <div class="note">
-            <p>Among other things, this means that:</p>
+          <p>
+            It is RECOMMENDED that users be able to request that persistent data be forgotten on a per-origin basis, particularly as part of a "Forget about this site" feature that forgets cookies [[!COOKIES]], databases, etc. associated with a particular site in an operation that is sufficiently atomic to prevent "cookie resurrection" type of recorrelation of a new identifier with the old by relying on another type of locally stored data that did not get cleared at the same time.
+          </p>
+        </section>
+        
```

Comment:
I believe you need another `</section>` here. Currently, Identifiers are nested in Persistent Data.